### PR TITLE
Catch both json and simplejson decode errors

### PR DESCRIPTION
**Pull request by jpmoral-factory**
Addresses #59 

Ugly, happy to receive suggestions.

### CODE REVIEW

**Review by jarekwg — CHANGES_REQUESTED** [review]
As mentioned in #59, I don't want to have direct reference to simplejson in this package, however, we could go with a solution like this one https://github.com/felix-hilden/tekore/pull/62/files
Turns out both `json.JSONDecodeError` and `simplejson.JSONDecodeError` are subclasses of `ValueError`.

File: tests/test_endpoints.py [hunk 1]
```diff
@@ -77,7 +76,7 @@ def test_json_error(self, mock_request):
         mock_request.return_value.status_code = 200
 
         def response_json():
-            raise JSONDecodeError('Some error message', '', 0)
+            raise ValueError
```
Comment (jarekwg) [inline tests/test_endpoints.py:79]: I think I'd still raise the JSONDecodeError here, but leave the lines below to be expecting ValueError.

Comment (jpmoral-factory) [reply tests/test_endpoints.py:79]: Ah, yeah, that makes sense.

**Comment by jarekwg** [comment]
Awesome, thanks!

**Comment by jarekwg** [comment]
Out in [1.2.14](https://github.com/uptick/pymyob/releases/tag/1.2.14).

**Pull request merged**